cli.js

```diff
@@ -170,6 +171,56 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   }, async (argv) => {
     await handleCommand(argv, listHandler);
   })
+  .command('triggers', 'Manages triggers', yargs0 => yargs0
+      .demandCommand()
+      .command('upsert', 'Creates a new trigger or updates an existing one', (yargs1) => {
```

Comment:
not sure about the name upsert.
i think it's better than either create up update, yet it may be confusing.

Reply:
lets try to think about a better name, maybe @rylandg or @avnerbraverman have a suggestion.
Leaving it as is for now

Reply:
Ack

Reply:
Setup / apply / deploy are good options

Reply:
@arikmaor please rename to one of the options above, I like `setup`.

Reply:
changed to setup